stored_queries/GO/GO_get_terms_from_ws_obj.yaml

```diff
@@ -0,0 +1,40 @@
+# Get the terms from a workspace object reference
+
+name: GO_get_terms_from_ws_obj
+params:
+  type: object
+  required: [obj_ref, ts]
+  properties:
+    obj_ref:
+      type: string
+      title: Workspace versioned object reference
+    limit:
+      type: integer
+      title: Maximum result limit
+      default: 20
+      maximum: 1000
+    offset:
+      type: integer
+      title: Result offset for pagination
+      default: 0
+      maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: ws_genome_features, GO_terms
+query: |
+  FOR o in ws_object_version
+    FILTER o._key == @obj_ref
+    FILTER o.is_public
```

Comment:
Change this to `v.is_public OR v.workspace_id IN ws_ids`. The variable `ws_ids` is automatically set to the api to a list of private workspaces that the user has access to